packages/react-sdk/src/tree/session-storage-polyfill.tsx

```diff
@@ -0,0 +1,35 @@
+// This is a temporary work around for https://github.com/remix-run/remix/issues/3659
+
+// The code is based on this discussion https://bugs.chromium.org/p/chromium/issues/detail?id=357625
+const polyfill = `(function () {
```

Comment:
its such a weird way to do the polyfill ... and then render it specifically before scrollrestoration over react ...
can't it just be imported as a normal script, before anything else runs?

Reply:
we will have the same issue if we try access sessionStorage in some script if it runs before this

Reply:
> render it specifically before scrollrestoration

The fact that it goes before `<ScrollRestoration>` is accidental, I just needed to put it before all scripts.

> can't it just be imported as a normal script

Not sure putting it in something like `entry.client.tsx` will work, as it'll probably go after `<ScrollRestoration>` in that case, but I can try.